core/runtime/jsruntime.rs

```diff
@@ -824,6 +845,14 @@ impl JsRuntime {
     };
     op_state.borrow_mut().put(isolate_ptr);
 
+    let mut fast_infos = Vec::new();
+    for op_ctx in &*op_ctxs {
+      fast_infos.push(FastInfo {
+        fn_info: op_ctx.fast_fn_c_info,
+        fn_sig: op_ctx.fast_fn_signature,
+      });
+    }
```

Comment:
Pre-allocate the vector, otherwise we're gonna hit several resizes when initializing ops in `deno`